structures/withTags.js

```diff
@@ -1,29 +1,18 @@
-const {
-  type,
-  optional,
-  array,
-  enums,
-  refine,
-} = require('superstruct');
+const { type, optional, array, enums, number, refine } = require('superstruct')
+
+exports.withTags = (config = {}) => {
+  const authorizedTags = Array.isArray(config?.authorizedTags) ? config.authorizedTags : []
+
+  const tagsStruct = array(enums(authorizedTags))
+
+  // If maxTags is defined, refine the tags structure to ensure its length doesn't exceed maxTags
+  const refinedTagsStruct =
+    config.maxTags && typeof config.maxTags === 'number'
```

Comment:
```suggestion
    config?.maxTags && typeof config.maxTags === 'number'
```
or `config && typeof config.maxTags === 'number'` for the same reason

Reply:
Done